apps/server/src/treematch/treematch.service.ts

```
@@ -0,0 +1,496 @@
+  // The flag lives in TTC, which also enforces its own rule (it refuses to
+  // change the flag of a fully allocated contribution). Note that the TTC
+  // endpoint is not project-scoped, so this cannot verify the contribution
+  // belongs to the project in the path.
+  async setContributionIgnore(
+    ttcContributionId: number,
+    dto: SetContributionIgnoreDto,
+  ) {
+    return this.ttcClient.setIgnore(ttcContributionId, {
+      ignored: dto.ignored,
+      ...(dto.ignored && dto.reason ? { reason: dto.reason } : {}),
+    });
```

ESLint

[error] Delete `;`

(prettier/prettier)